docs/examples.rst

```diff
@@ -0,0 +1,368 @@
+    
+    [weight]
+    _Help = Weighting options
+    column = None                           # Weight column to use.
+
+    [flags]
+    _Help = General flagging options
+    apply = -cubical                        # Ignore existing cubical flags.
+
+    [sol]
+    _Help = Solution options which apply at the solver level
+    jones = G                               # Term/s to solve for.
+                                            # Corresponding section/s below.
+    delta-g = 1e-06                         # Stopping criteria on gains.
+    delta-chi = 1e-06                       # Stopping criteria on chi-squared.
+    term-iters = 20                         # Iteration recipe.
+                                            # Overrides individual term settings.
+
+    [dist]
+    _Help = Parallelization and distribution options
+    ncpu = 4                                # Number of processes to use.
+                                            # Implicitly controls memory footprint.
+    max-chunks = 12                         # Maximum number of simultaneous chunks.
+                                            # Ideally >= ncpu.
+    min-chunks = 12                         # Minimum number of simultaneous chunks.
+                                            # Ideally = ncpu.
+    safe = 1.0                              # Fraction of total memory CubiCal may use.
+                                            # Will crash deliberately if exceeded.
+
+    [g]
+    _Help = Options for G-Jones term
+    label = G                               # This term's name.
+                                            # Must match [sol] section's jones option.
+    type = phase-diag                       # Solve for a diagonal phase-only gain. 
+    time-int = 1                            # Time interval for solution.
+    freq-int = 1                            # Frequency interval for solution.
+    max-iter = 20                           # Maximum number of iterations.
+                                            # May be ignored if [sol] term-iters is set.
+    _Templated = True                       # Other parameters polulated from defaults.
+                                            # Can be safely ignored.
+
+Phase and amplitude selfcal
+############################
+
+The following performs diagonal, phase and amplitude selfcal using a Tigger
+.lsm sky model and produces corrected data. Note that using sky models requires 
+CubiCal to be installed with lsm-support.
+
+.. code-block::
+
+    [data]
+    _Help = Visibility data options
+    ms = ms.ms                              # Your measurement set name.
+    column = DATA                           # Column in which the data lives.
+    time-chunk = 100                        # Number of time slots in a chunk. 
+                                            # Implicitly controls memory footprint.
+    freq-chunk = 64                         # Number of frequencies in a chunk. 
```

Comment:
"Number of frequency channels in a chunk" ?

Reply:
I've never actually used it in this way, but can't you also specify units here (`Hz`, and and also `s` for time chunks / solution intervals)? Current or suggested help string wouldn't be applicable if so...

Reply:
It does work both ways, but I wanted to strip out as many options/explanations as possible. Will go with the simpler integers for now. These examples don't have to be set in stone, so if users trip up on this we can change/update them.